app/lib/actions/katello/upstream_subscriptions/remove_entitlement.rb

```diff
@@ -6,10 +6,13 @@ class RemoveEntitlement < Actions::Base
 
         input_format do
           param :entitlement_id
+          param :sub_name
         end
 
         def run
           output[:response] = ::Katello::Resources::Candlepin::UpstreamConsumer.remove_entitlement(input[:entitlement_id])
+        rescue ::Katello::Errors::UpstreamEntitlementGone
+          output[:response] = "entitlement (subscription) with name \"%s\" has already been deleted" % input[:sub_name]
```

Comment:
What do you think about reworking this message? "%s has already been deleted". My thought is that it's a little more concise. We already have the context of what object we are dealing with when we are looking at the message in the dynflow console (::Remove**Entitlement**). I don't feel too strongly about it, so up to you!

Reply:
Oh! In any case this string needs to be surrounded by _("") so that it can be translated

Reply:
Yeah I was thinking it's a bit long.  I think shortening it works fine.

Reply:
Pushed the change here!